app/services/mlops/hopsworks_service.py

```
@@ -0,0 +1,708 @@
+            logger.info(f"Created feature view: {name} v{version}")
+            return feature_view
+
+        except Exception as e:
+            logger.error(f"Error creating feature view: {e}")
+            return None
+
+    async def get_training_data(
+        self,
+        feature_view_name: str,
+        version: int = 1,
+        start_time: Optional[datetime] = None,
```

Ruff (0.13.3)

Unused method argument: `start_time`

(ARG002)